### PR TITLE
Blank column

### PR DESCRIPTION
**Pull request by mXaln**

### CODE REVIEW

File: USFMToolsSharp.Renderers.HTML/HTMLConfig.cs [hunk 1]
```diff
@@ -17,13 +18,15 @@ public HTMLConfig()
             divClasses = new List<string>();
             separateChapters = false;
             separateVerses = false;
+            blankColumn = false;
             partialHTML = false;
         }
-        public HTMLConfig(List<string> divClasses, bool separateChapters = false, bool partialHTML = false, bool separateVerses=false)
+        public HTMLConfig(List<string> divClasses, bool separateChapters = false, bool partialHTML = false, bool blankColumn = false, bool separateVerses=false)
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.HTML/HTMLConfig.cs]: You should put the new column at the end of the list.

Comment (mXaln) [reply USFMToolsSharp.Renderers.HTML/HTMLConfig.cs]: @rbnswartz Done

File: USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs [hunk 2]
```diff
@@ -67,10 +67,22 @@ public string Render(USFMDocument input)
                 output.AppendLine($"<div class=\"{class_name}\">");
             }
 
+            // Will add a table with two columns where the second column will be blank
+            if(ConfigurationHTML.blankColumn)
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs]: Can you try to make this in CSS?

Comment (mXaln) [reply USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs]: @rbnswartz Do you mean avoid using table?

**Comment by rbnswartz** [comment]
I don't know if this is the way that we really want to do this but it is the easiest at the moment.

**Pull request merged**